core/src/main/java/io/questdb/std/Chars.java

```diff
@@ -130,25 +130,67 @@ public static boolean contains(CharSequence sequence, CharSequence term) {
     }
 
     public static int indexOf(CharSequence sequence, int sequenceLo, int sequenceHi, CharSequence term) {
+        return indexOf(sequence, sequenceLo, sequenceHi, term, 1);
+    }
+
+    public static int indexOf(CharSequence sequence, int sequenceLo, int sequenceHi, CharSequence term, int occurrence) {
```

Comment:
Looks like these changes broke `HttpHeaderParserTest#testWrite()` and `testSplitWrite()` tests. They use `Chars#contains()` as a part of the assertions. Could you check?

Reply:
Sorry, I made a mistake in the algorithm, I've fixed it, thanks! Sorry for having you review so many times

Reply:
Yes, I've checked the latest commit. Thanks for the quick fix!

Reply:
@daniel-shuy great job pushing your implementation successfully.